### PR TITLE
Update the Tutorials timeline for 17 weeks

### PR DESCRIPTION
**Pull request by sbis04**
Have diluted the timeline to `17 weeks` and updated the **Tutorials** blog post.

### CODE REVIEW

**Comment by Rohitesh-Kumar-Jain** [comment]
Thanks for updating, seems good to me.

File: jekyll/_posts/2021-05-11-gsod-2021-team-proposal-tutorials.md [hunk 1]
```diff
@@ -71,6 +71,10 @@ The overall structure of the Tutorials will be as follows:
   
   Here we can have all the links for each of the pages present in the Tutorials section along with a brief description. This will help the user to easily navigate through the Docs.
 
+* **Installation**
+  
+  A guide for installing the development tools required for running Wechaty bot on any user system.
+
 * **Quick start**
```
Comment (Rohitesh-Kumar-Jain) [inline jekyll/_posts/2021-05-11-gsod-2021-team-proposal-tutorials.md:78]: LGTM timeline

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Comment by proudofsimin** [comment]
It looks great! Thank you for your effort!

**Review by huan — APPROVED** [review]
LGTM

Please get approvals from all volunteers and your team mates to make this PR fulfill the requirements of merging.

Thank you very much!

**Comment by sbis04** [comment]
@proudofsimin & @shwetalsoni if this PR is alright can you please approve it, so that it fulfills the requirements of merging.

**Review by shwetalsoni — APPROVED** [review]

**Comment by sbis04** [comment]
As @proudofsimin had requested to list down the changes in the updated timeline for easier understanding, there are as follows:

* We have added a section called **Installation** page to our overall tutorial structure, and added it under our week 2 tasks.
* Instead of keeping the "Usage with Docker" and "Usage with Heroku" under **Tutorials**, it is to be included under **How-to guides** (only links for those pages will be under tutorials so that users can quickly navigate to them if they are searching for it, this resolves the conflict between the tutorials and the how-to guide sections).
* Remaining things are almost the same, just diluted the timeline to 17 weeks, @proudofsimin you can update them on your docs (where you are tracking the completion of tasks w.r.t. PRs).

We have also kept our [Team proposal docs](https://docs.google.com/document/d/16MDUHguYxlHXxXH830I1OskEFml8_tdUzhThXYUqbn8/edit?usp=sharing) updated with these changes, and if any further changes are required we will update it in this Docs.

**Review by proudofsimin — APPROVED** [review]

**Comment by Rohitesh-Kumar-Jain** [comment]
ping @huan

**Comment by Rohitesh-Kumar-Jain** [comment]
@huan can we merge this pls? It will be great if the updated proposal is merged : )

**Review by lijiarui — APPROVED** [review]

**Review by huan — APPROVED** [review]

**Pull request merged**